README.md

```diff
@@ -121,13 +121,13 @@ This error shows the request is missing a required property `name` from the HTTP
 
 #### Server Validation
 
-OpenAPI lets API spec authors make only certain servers available, and they also allow certain operations to be restricted to certain servers. Make sure the server URL you plan to use is a valid server this the particular operation you are attempting.
-by providing a `__server` query param.
+OpenAPI lets API spec authors make only certain servers available, and they also allow certain
```

Comment:
I'd really suggest not to put newlines in Markdown documents and try to rely as much as possible on your editor wrapper capabilities — otherwise everybody will go like crazy on this.

Reply:
Man this is something I've _always_ done. Got a vscode extension for it and everything. It's for folks who use shit like Vim without proper extensions set up, etc.

Reply:
I'm ok with this but just keep in mind, again that

1. Not everybody uses VSCode (I do not)
2. Vim is not shit

Reply:
Haha right Vim is amazing with plugins enabled. Hard wrapping lines like this makes it a lot more usable for those who havent set up soft wrapping.